install/v2plugin/README.md

```diff
@@ -30,31 +32,104 @@ dbg_flag      : To enable debug mode, set to '-debug'         : ""
 ### docker store
 Docker certified contiv plugin is avaliable on [Docker Store](https://store.docker.com/plugins/803eecee-0780-401a-a454-e9523ccf86b3?tab=description).
 ```
-docker plugin install store/contiv/v2plugin:1.0.0-beta.3 iflist=eth1,eth2
+docker plugin install store/contiv/v2plugin:1.0.0 iflist=eth1,eth2
 ```
 ### docker hub
 Contiv plugin released from contiv repo is also pushed to docker hub. iflist has the list of data interfaces used for vlan networks in contiv.
 ```
-docker plugin install contiv/v2plugin:1.0.0-beta.3 iflist=eth1,eth2
+docker plugin install contiv/v2plugin:1.0.0 iflist=eth1,eth2
 ```
 ### vagrant dev/demo setup
 To create a plugin from [contiv repo](https://github.com/contiv/netplugin), enable v2plugin and run docker in swarm-mode, use the Makefile target demo-v2plugin
 ```
 make demo-v2plugin
 ```
 
-## Contiv plugin-modes
+## Contiv plugin-roles
 Contiv plugin runs both netplugin and netmaster by default. Contiv v2plugin can be run with only netplugin by setting the plugin_role to slave.
 ```
-docker plugin install contiv/v2plugin:1.0.0-beta.3 iflist=eth1,eth2 plugin_role=slave
+docker plugin install contiv/v2plugin:1.0.0 iflist=eth1,eth2 plugin_role=slave
 ```
 
-## Contiv plugin workflow
+## Contiv plugin compose workflow (recommended and default for v2plugin)
   1. Etcd cluster should be brought up on the hosts on localhost:2379. If a different port (or Consul) is used, cluster-store option needs to be specified in the plugin install command.
 
+  2. Bring up Docker Swarm-mode
+  ```
+  # on manager node init swarm-mode
+  docker swarm init --advertise-addr 192.168.2.10:2377
+
+  # get the join-token from master node
+  docker swarm join-token worker -q
+
+  # on worker nodes, use the token to join swarm
+  docker swarm join --token SWMTKN-1-4qgg20vkzhc3jhc765k5x0coyriggkdvw1t7fmbiimqguagqr7-8um9goip0d03yqmmrb4c4fh1j 192.168.2.10:2377  
+  ```  
+  3. Install contiv v2plugin
+  ```
+  # on swarm manager node install plugin with 'master' role
+  docker plugin install contiv/v2plugin:1.0.0 plugin_role=master iflist=<data ifs used for vlan networks>
```

Comment:
auth_proxy is run as a different container by contiv/installer. Netmaster will be part of the plugin